### PR TITLE
[Update] Host a Static Site using Linode Object Storage: (Optional) Next Steps Edit

### LINKED ISSUE #2842: Object storage: imprecisions on documentation of hosting static websites and custom domains
Hi,

I think there are a few imprecisions on section [(Optional) Next Steps](https://www.linode.com/docs/platform/object-storage/host-static-site-object-storage/#optional-next-steps):

1) It misses the mention that for custom domains to work, the name of the bucket needs to be equal to the name of the custom domain (e.g. for **www.my-domain.com** the name of the bucket needs to be "**www.my-domain.com**", otherwise it won't work).

2) there's a mention to adding a CNAME to the bucket "URL". But the correct is to say to point the CNAME do the hostname of the bucket (e.g. CNAME www.my-bucket.com pointing to www.my-bucket.com.website-us-east-1.linodeobjects.com ). And explain it better with an example because it's not obvious to figure out the string to which the CNAME should point to.

3) It misses the mention that when using a custom domain the SSL functionality doesn't work for that domain, at the moment (i.e. will display a certificate warning).

### PR DESCRIPTION
**Pull request by bagasme**
* Clarify CNAME addition for custom domain

* Add note that bucket name must match domain name

* Add SSL incompatibility note for custom domain

This PR will close #2842.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 0ceb779d316324e4af21ed977a4ba40eaef82cb5

https://deploy-preview-2851--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by bagasme** [comment]
@andystevensname Why do not merge?

**Comment by Guaris** [comment]
@andystevensname

File: docs/platform/object-storage/host-static-site-object-storage/index.md [hunk 1]
```diff
@@ -293,19 +293,18 @@ It may take a minute or two after your s3cmd sync completes for the page to appe
 
 ## (Optional) Next Steps
 
-After uploading your static site to Linode Object Storage, you may want to use a custom domain for your site. To do this, you can add a CNAME entry to your domain's DNS records that aliases it to your Object Storage bucket's website URL. For example, if you have `www.mydomain.tld`, your CNAME entry will look like:
+After uploading your static site to Linode Object Storage, you may want to use a custom primary domain for your site. To do this, you can add a CNAME entry to your domain's DNS records that aliases it to your Object Storage bucket's website URL. For example, if you have `www.mydomain.tld`, your CNAME entry will look like:
```
Comment (bagasme) [inline docs/platform/object-storage/host-static-site-object-storage/index.md]: Oh dear, primary custom subdomain?

File: docs/platform/object-storage/host-static-site-object-storage/index.md [hunk 2]
```diff
@@ -293,19 +293,18 @@ It may take a minute or two after your s3cmd sync completes for the page to appe
...
-{{< note >}}
-For custom domains to work with Linode Object Storage, ensure that the name of your bucket is same as your custom domain. So if you wish to serve your site as `www.mydomain.tld`, the bucket name MUST be `www.mydomain.tld`.
-{{< /note >}}
+        subdomain.mydomain.tld										CNAME	www.my-new-bucket.us-east-1.linodeobjects.com
+
+To learn about managing DNS records on Linode, see the [DNS Manager](/docs/platform/manager/dns-manager/) and [DNS Records: An Introduction](/docs/networking/dns/dns-records-an-introduction/) guides.
 
 {{< note >}}
-SSL functionality doesn't work when using custom domain, since the certificate only valid for website endpoint, not custom domain. You will see certificate warning thrown if you proceed to access your site via `https`.
+SSL functionality does not work when using a custom domain, since the certificate is only valid for the website endpoint, not your custom domain. You will see a certificate warning thrown if you proceed to access your custom domain via `https`.
```
Comment (bagasme) [inline docs/platform/object-storage/host-static-site-object-storage/index.md]: OK for grammatical correction

**Comment by andystevensname** [comment]
This just needs to be tested before it goes into develop, which is why I marked it as Do Not Merge. I've removed the label.

**Pull request merged**